ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+   var body: some View {
+       VStack(spacing: 15) {
+           NavigationStack{
+               VStack(alignment: .leading, spacing: 15){
+                   Text("Task Title")
+                       .multilineTextAlignment(.leading)
+                   TextField("Insert Title", text: $createTaskViewModel.titleInput)
+                       .padding(.leading, 10.0)
+                     .textInputAutocapitalization(.words)
+                     .frame(width: 390.0, height: 40)
```

Comment:
we should not hardcode the width since there are various screen sizes by hardcoding the width we may get either too small text fields on big screens or too big text fields on small screens